src/Nordea/Components/InformationDetails.elm

```diff
@@ -60,6 +82,66 @@ card attrs children title =
             ]
 
 
+collapsibleCard :
+    { attrs : List (Attribute msg)
+    , title : String
+    , emphasisedText : String
```

Comment:
Instead of taking a string, what do you think about taking html? This makes it possible to slightly customize the content. For example, wrap the text in a span to change appearance, add accessibility attributes, or have a multiline text, etc.
```suggestion
    , emphasisedText : Html msg
```

Reply:
Do we still want the emphasisedText to have the fade in/out transition in all cases or should that be passed within the emphasisedText : Html msg?

Reply:
I would say let it always have it. We can add a parameter for it later if needed